### PR TITLE
Generate GH-pages HTML files again (alternative approach)

### LINKED ISSUE #57: Data browser web app has vanished
https://solid.github.io/mashlib/dist/browse.html
used to be an online version of the databrowser,
which is quite handy for inspecting remote data. 
Like many RDF viewers on the web.

It has been removed in that dist is cleaned by out NPM run clean I suspect.

### PR DESCRIPTION
**Pull request by Vinnl**
This PR is an alternative approach to #61, incorporating suggestions made by @RubenVerborgh: https://github.com/solid/mashlib/pull/61#issuecomment-522943381

I personally prefer the approach in #61 for the reasons described in my responses there (particularly https://github.com/solid/mashlib/pull/61#issuecomment-522947558), but if consensus is that Ruben's approach is better, here's the PR to implement that :)

Fixes #57.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "dist"
   ],
   "scripts": {
-    "build": "npm run build:dev && npm run build:prod",
+    "build": "npm run build:dev && npm run build:prod && cp static/* dist",
```
Comment (RubenVerborgh) [inline package.json]: I suggest to make them part of the webpack process, so they are also served on dev.

**Comment by Vinnl** [comment]
Note: I just noticed I'd reverted too much. The diff is a bit smaller now.

File: package.json (hunk 1 above)
Comment (Vinnl) [reply package.json]: See 9e5aac2e884ebdbb337d6d7d26b2851f181a9f3c.

**Comment by timbl** [comment]
Lets go with this

**Pull request merged**